build.gradle

```diff
@@ -52,16 +52,16 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-security'
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 
-	implementation 'eu.xenit.contentcloud.thunx:thunx-spring:0.1.0'
-	implementation 'eu.xenit.contentcloud.thunx:thunx-pdp-opa:0.1.0'
+	implementation 'eu.xenit.contentcloud.thunx:thunx-spring:0.3.0'
+	implementation 'eu.xenit.contentcloud.thunx:thunx-pdp-opa:0.3.0'
 
 	compileOnly 'org.projectlombok:lombok'
 	annotationProcessor 'org.projectlombok:lombok'
 
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testImplementation 'org.springframework.security:spring-security-test'
 	testImplementation 'com.jayway.restassured:rest-assured:2.9.0'
-	testImplementation platform('org.testcontainers:testcontainers-bom:1.16.2')
+	testImplementation platform('org.testcontainers:testcontainers-bom:1.16.3')
```

Comment:
`testcontainers` version is since Spring Boot 2.6.x managed by the spring boot platform